omni_detector:
  ros__parameters:
    use_sensor_data_qos: true
    debug_mode: true
    camera_num: 1

    omni:
      camera0_name: 'front_industrial_camera'
      subscribe_compressed: false

      # Model
      model_path: $(find-pkg-share armor_detector_openvino)/model/opt-1208-001.onnx
      device_type: 'AUTO'
      confidence_threshold: 0.25
      top_k: 128
      nms_threshold: 0.3
